app/views/conversations/_messages.haml

```diff
@@ -0,0 +1,15 @@
+.stream
+  = render :partial => 'message', :collection => conversation.messages
```

Comment:
Please use the Ruby 1.9 hash syntax, `render partial: "message", collection: conversation.messages`